src/app.rs

```diff
@@ -21,7 +18,7 @@ use types::{RoutesFn, SinkFn, UpdateFn, ViewFn, WindowEventsFn};
 use wasm_bindgen::closure::Closure;
 use web_sys::Element;
 
-pub mod builder;
+pub(crate) mod builder;
```

Comment:
Don't play with it, kill it! :D

Reply:
That's the plan. There are some modules underneath `builder` that breaks other code. I don't want to pull out the dependencies out from under them. I'll cull the the implementations of `builder/*.rs` line-by-line, and at the end, no builder.